.nuspec/Microsoft.Maui.Controls.MultiTargeting.targets

```diff
@@ -79,11 +79,6 @@
     <MonoAndroidAssetsPrefix>$(AndroidProjectFolder)Assets</MonoAndroidAssetsPrefix>
   </PropertyGroup>
 
-  <!-- .NET Standard or .NET 6 -->
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'net$(_MauiDotNetVersion)' ">
-    <DefineConstants>$(DefineConstants);NETSTANDARD</DefineConstants>
-  </PropertyGroup>
```

Comment:
If the TFM was `net6.0` then we defined `NETSTANDARD`. This is no longer the case as it is very confusing.